src/Microsoft.Health.Dicom.Functions/Registration/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using EnsureThat;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Microsoft.Health.Dicom.Core.Registration;
+using Microsoft.Health.Dicom.Operations.Functions.Configs;
+using Newtonsoft.Json.Converters;
+
+namespace Microsoft.Health.Dicom.Operations.Functions.Registration
+{
+    public static class ServiceCollectionExtensions
+    {
+        /// <summary>
+        /// Add services for DICOM functions.
+        /// </summary>
+        /// <param name="services">The DICOM function builder instance.</param>
+        /// <param name="configuration">The configuration</param>
+        /// <returns>The DICOM function builder instance.</returns>
+        public static IDicomFunctionsBuilder AddDicomFunctions(this IServiceCollection services, IConfiguration configuration)
+        {
+            EnsureArg.IsNotNull(services, nameof(services));
+            DicomFunctionsConfiguration dicomOperationsConfig = new DicomFunctionsConfiguration();
+            configuration?.GetSection(DicomFunctionsConfiguration.SectionName).Bind(dicomOperationsConfig);
```

Comment:
nit: we can use `Get<T>` instead of `Bind(object)`.

Eg.
```csharp
configuration.GetSection(DicomFunctionsConfiguration.SectionName).Get<DicomFunctionsConfiguration>();
```